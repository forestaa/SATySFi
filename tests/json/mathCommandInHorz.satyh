let-math \f x = ${2} in {\f{1}}
