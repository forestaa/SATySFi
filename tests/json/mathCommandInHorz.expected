{
  "file": "mathcommandinhorz.satyh",
  "range": {
    "start": { "line": 1, "character": 25 },
    "end": { "line": 1, "character": 27 }
  },
  "message": "! [Type Error] a math command is used as an inline command."
}
